### ISSUE TITLE
Extend the contributing readme section with the rulles to bare in mind

### ISSUE THREAD

**Issue by denis-konovalyenko** [amihaiemil/zold-java-client#13]
It would be better to have a message like that:
"We don't have a QAs on the projects but, we should still try to respect [the QA rules](http://www.zerocracy.com/policy.html#42) as much as possible. That means that we should not block PRs for small stuff like commit messages and so on."

**Comment by 0crat** [comment]
@amihaiemil[/z](https://www.0crat.com/u/amihaiemil) please, pay attention to this issue

**Comment by amihaiemil** [comment]
@denis-konovalyenko Right, but let's give more context/details. I think  he paragraph should be: 

"This project is managed by [Zerocracy](http://www.zerocracy.com/), see the [Policy](http://www.zerocracy.com/policy.html) for more details.

Note that we do not have Zerocracy QAs, yet we still try to adhere to the [QA rules](http://www.zerocracy.com/policy.html#42) as much as possible (we won't block any PRs for cosmetic stuff such as commit messages, though)."

**Comment by denis-konovalyenko** [comment]
@amihaiemil, agree, that sounds better.

**Comment by 0crat** [comment]
Job [#13](https://github.com/amihaiemil/zold-java-client/issues/13) is now in scope, role is `DEV`

**Comment by 0crat** [comment]
Bug was reported, see [§29](http://www.zerocracy.com/policy.html#29): +15 point(s) just awarded to @denis-konovalyenko[/z](https://www.0crat.com/u/denis-konovalyenko)

**Comment by 0crat** [comment]
Job `gh:amihaiemil/zold-java-client#13` already assigned to @paulodamaso, can't assign to @paulodamaso

**Comment by 0crat** [comment]
The job [#13](https://github.com/amihaiemil/zold-java-client/issues/13) is now out of scope

**Comment by 0crat** [comment]
Order was finished: +35 point(s) just awarded to @paulodamaso[/z](https://www.0crat.com/u/paulodamaso)
